cmd/apiserver-boot/boot/glide.go

```diff
@@ -40,7 +40,7 @@ func AddGlideInstallCmd(cmd *cobra.Command) {
 }
 
 func fetchGlide() {
-	c := exec.Command("glide", "install", "--strip-vendor")
+	c := exec.Command("glide", "install", "--strip-vendor", "--strip-vcs")
```

Comment:
hm.  good catch.  the reason I added this is that it is required with `strip-vendor` in the glide version that is installed by apt-get get.  :(  Maybe instead we should just do a version check for `glide` and force users to install a version gt X